staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -294,6 +294,9 @@ type VirtualMachineInstanceStatus struct {
 	// Memory shows various informations about the VirtualMachine memory.
 	// +optional
 	Memory *MemoryStatus `json:"memory,omitempty"`
+
+	// BackendStorage contains information about the PVC used to store the backend state of the VM
+	BackendStorage *BackendStorageStatus `json:"backendStorage,omitempty"`
```

Comment:
Maybe we could include this in the [`VolumeStatus`](https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/api/core/v1/types.go#L254)

Reply:
It is still a PVC and in the end it is storage used by the VM. So, I think it fits there.